docs/programming_guide/provisioning_system.rst

```diff
@@ -283,6 +283,62 @@ A new builder to write 'gateway.conf' can be implemented as follows (for referen
                   f.write(f"name = {gw.name}\n")
                   f.write(f"port = {port}\n")
 
+.. _distribution_builder:
+
+Case 4: adding a builder for enabling the creation of zip archives for the startup kits
+---------------------------------------------------------------------------------------
+DistributionBuilder was included in NVIDIA FLARE before version 2.2.1 but has been removed from the
+default builders. You can add it as a builder in project.yml if you want to zip the startup kits::
```

Comment:
Why copy the class code here? Would be better to show how it looks in the .yml file.

Reply:
Since this code has been removed in 2.2.1, the user would have to include it in order to make things work. I added an example for what needs to be included in the .yml file too.

Reply:
I see.